HACKING.md

```diff
@@ -0,0 +1,29 @@
+
+#Hacking On The Manifesto
+
+###Background
+
+The manifesto acts as a sort of code of conduct for developers down with the Namespace App Club movement. While we won't
```

Comment:
I agree with @benforeva, let's stick with just Namespace as the identifier. App Club doesn't have a strict definition but I can see it useful in a definition. As a solution we use a hyphen/comma like this: Namespace - Developer App Club (not those particular words necessarily, just the concept)